.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Order is important. The last matching pattern has the most precedence.
+
+* @chef/jex-team
```

Comment:
should this be `releng-team` now?

Reply:
No it should not :)

Reply:
No - we're going to keep it jex-team since the repo doesn't really matter to the docs-team.